habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -279,6 +296,80 @@ def calculate_walk_pose(
         self.obj_transform_base = obj_transform_base @ rot_offset
         self.joint_pose = joint_pose
 
+    def translate_and_rotate_with_gait(
```

Comment:
This function is not being called anywhere. It probably should be the default from now on.

It would be worth replacing all calls to `calculate_walk_pose_directional` and deprecating it.